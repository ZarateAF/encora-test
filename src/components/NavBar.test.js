import renderer from "react-test-renderer";
import { NavBar } from "./";

it("renders correctly", () => {
  const tree = renderer
    .create(<NavBar title={"title"} onClick={(a) => a}></NavBar>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
