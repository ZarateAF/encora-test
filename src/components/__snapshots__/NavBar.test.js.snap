// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="navbar"
>
  <div
    className="home"
    onClick={[Function]}
  >
    HOME
  </div>
  <h1>
    title
  </h1>
</div>
`;
